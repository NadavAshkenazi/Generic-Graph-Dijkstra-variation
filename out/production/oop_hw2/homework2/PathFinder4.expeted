created graph A
created node n1 with cost 1
created node n2 with cost 1
created node n3 with cost 1
created node n4 with cost 1
created node n5 with cost 1
added node n1 to A
added node n2 to A
added node n3 to A
added node n4 to A
added node n5 to A
added edge from n3 to n1 in A
added edge from n1 to n4 in A
added edge from n1 to n5 in A
added edge from n2 to n5 in A
added edge from n3 to n2 in A
added edge from n2 to n4 in A
no path found in A