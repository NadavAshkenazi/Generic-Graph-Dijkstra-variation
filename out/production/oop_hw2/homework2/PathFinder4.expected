created graph A
created node n1 with cost 1
added node n1 to A
shortest path in A: n1
